bigbluebutton-html5/private/locales/en.json

```diff
@@ -21,6 +21,9 @@
     "app.note.title": "Shared Notes",
     "app.note.label": "Note",
     "app.note.hideNoteLabel": "Hide note",
+    "app.user.activityCheck": "User activity check",
```

Comment:
We call it `inactivity` everywhere but the locales and the component. Consider keeping it as `inactivity`

Reply:
I did this way to follow the redis messages: `UserInactivityInspectMsg ` and `UserActivitySignCmdMsg `.